sdk/eventhub/Azure.Messaging.EventHubs/stress/src/Program.cs

```diff
@@ -79,118 +68,88 @@ private static async Task RunOptions(Options opts)
-        {
-            // Get the needed resources for the buffered producer test: an event hub
-            var eventHubName = String.Empty;
-            environment.TryGetValue(EnvironmentVariables.EventHubBurstBufferedProducerTest, out eventHubName);
-            eventHubName = PromptForResources("Event Hub", "Burst Buffered Producer Test", eventHubName, opts.Interactive);
-            testConfiguration.EventHub = eventHubName;
-
-            roleConfiguration.TestScenarioRoles.TryGetValue(RoleConfiguration.BurstBufferedProducerTest, out var roleList);
-            testScenarioTasks.Add(RunScenario(roleList, testConfiguration, roleConfiguration, metrics, opts.Role, RoleConfiguration.BurstBufferedProducerTest, cancellationSource.Token));
-        }
-
-        // If running the Processor Test scenario, gather resources and add a scenario run to the task list.
-        if (opts.Test == RoleConfiguration.EventProcessorTest || opts.Test == RoleConfiguration.EventProcessorTestShort || opts.All)
-        {
-            // Get the needed resources for the processor test: an event hub, storage account, and blob container name
-            var eventHubName = String.Empty;
-            var storageBlob = String.Empty;
-            var storageConnectionString = String.Empty;
-            environment.TryGetValue(EnvironmentVariables.EventHubProcessorTest, out eventHubName);
-            eventHubName = PromptForResources("Event Hub", RoleConfiguration.EventProcessorTest, eventHubName, opts.Interactive);
-            testConfiguration.EventHub = eventHubName;
-
-            environment.TryGetValue(EnvironmentVariables.StorageBlobProcessorTest, out storageBlob);
-            storageBlob = PromptForResources("Storage Blob Name", RoleConfiguration.EventProcessorTest, storageBlob, opts.Interactive);
-            testConfiguration.BlobContainer = storageBlob;
-
-            environment.TryGetValue(EnvironmentVariables.StorageAccountProcessorTest, out storageConnectionString);
-            storageConnectionString = PromptForResources("Storage Account Connection String", RoleConfiguration.EventProcessorTest, storageConnectionString, opts.Interactive);
-            testConfiguration.StorageConnectionString = storageConnectionString;
-
-            roleConfiguration.TestScenarioRoles.TryGetValue(RoleConfiguration.EventProcessorTest, out var roleList);
-            testScenarioTasks.Add(RunScenario(roleList, testConfiguration, roleConfiguration, metrics, opts.Role, RoleConfiguration.BurstBufferedProducerTest, cancellationSource.Token));
-        }
+        var metrics = new Metrics(appInsightsKey);
 
-        // Wait for all scenario runs to finish before returning.
-        await Task.WhenAll(testScenarioTasks).ConfigureAwait(false);
-    }
+        using var azureEventListener = new AzureEventSourceListener((args, level) => metrics.Client.TrackTrace($"EventWritten: {args.ToString()} Level: {level}."), EventLevel.Warning);
 
```

Comment:
Following up on thoughts in the `EventTracking` class - you could also consider creating the hasher here and passing it into the places it was used.